src/pages/Dashboard.js

```diff
@@ -51,9 +40,7 @@ const Dashboard = () => {
       <main>
         <Navbar></Navbar>
         <Search />
-        <Info />
-        <User />
-        <Repos />
+        {ifLoadingContent}
```

Comment:
You can add this logic directly in the markup here without saving it to the `ifLoadingContent` variable

Or if you decide to leave it as is, it's better to rename the variable to something more descriptive